packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -108,19 +112,18 @@ const updateAllSchema = z.object({
 });
 
 /** ---------------------------------------------------------------------------
- * POST /internal/patient/update-all
+ * POST /internal/patient/update-all/commonwell
  *
  * Triggers an update for all of a cx's patients without changing any
  * demographics. The point of this is to trigger an outbound XCPD from
  * CommonWell to Carequality so new patient links are formed.
  *
- *
  * @param req.query.cxId The customer ID.
  * @param req.body.patientIds The patient IDs to update (optional, defaults to all patients).
  * @return count of update failues, 0 if all successful
  */
 router.post(
-  "/update-all",
+  "/update-all/commonwell",
```

Comment:
assuming Postman was updated as well?

Reply:
Good catch, I was waiting for this to be merged, forgot to add to the release plan. Postman updated.